src/test/resources/application.properties

```diff
@@ -1,2 +1,4 @@
 AUTH_ISSUER=https://domain.tld/issuer
-GCP_JWK_PROVIDER_BASE_URL=https://gcpjwk.domain.tld/
\ No newline at end of file
+GCP_JWK_PROVIDER_BASE_URL=https://gcpjwk.domain.tld/
+SVC_AUTH_AUDIENCE=''
+GCP_PROJECT_WHITELIST=''
```

Comment:
I just noticed when doing the tests on my ticket, that the application properties env vars are also set in the `integration` directory in this repo.

The unit tests usually are usually very isolated and only include only the class under test or other few classes in the test context so it might be better to put your tests in the integration folder

Reply:
done!